Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -229,70 +81,31 @@ document title. This way you can link to a documents start page by:
 
     See :ref:`TYPO3 Explained <t3coreapi:start>`.
 
+If your manual has more pages then this one it must contain a table of content
+directive called :rst:`..  toctree::`. The `toctree` on the start page
+defines which pages will be displayed in the menu on the left of the rendered
```

Comment:
Would prefer "in the main navigation" instead "on the left" as "left" is not correct in mobile view.